apps/condo/migrations/20220721202935-0153_remove_ticket_placeclassifier_and_more.js

```diff
@@ -113,67 +113,67 @@ ALTER TABLE "TicketChange" DROP COLUMN "classifierRuleDisplayNameFrom" CASCADE;
 --
-ALTER TABLE "TicketChange" ADD COLUMN "placeClassifierIdFrom" uuid NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "placeClassifierIdFrom" uuid NULL;
 --
 -- Remove field placeClassifierDisplayNameTo from ticketchange
 --
-ALTER TABLE "TicketChange" ADD COLUMN "placeClassifierDisplayNameTo" text NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "placeClassifierDisplayNameTo" text NULL;
 --
 -- Remove field placeClassifierDisplayNameFrom from ticketchange
 --
-ALTER TABLE "TicketChange" ADD COLUMN "placeClassifierDisplayNameFrom" text NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "placeClassifierDisplayNameFrom" text NULL;
 --
 -- Remove field categoryClassifierIdTo from ticketchange
 --
-ALTER TABLE "TicketChange" ADD COLUMN "categoryClassifierIdTo" uuid NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "categoryClassifierIdTo" uuid NULL;
 --
 -- Remove field categoryClassifierIdFrom from ticketchange
 --
-ALTER TABLE "TicketChange" ADD COLUMN "categoryClassifierIdFrom" uuid NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "categoryClassifierIdFrom" uuid NULL;
 --
 -- Remove field categoryClassifierDisplayNameTo from ticketchange
 --
-ALTER TABLE "TicketChange" ADD COLUMN "categoryClassifierDisplayNameTo" text NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "categoryClassifierDisplayNameTo" text NULL;
 --
 -- Remove field categoryClassifierDisplayNameFrom from ticketchange
 --
-ALTER TABLE "TicketChange" ADD COLUMN "categoryClassifierDisplayNameFrom" text NULL;
+ALTER TABLE "TicketChange" ADD COLUMN IF NOT EXISTS "categoryClassifierDisplayNameFrom" text NULL;
 --
 -- Remove field problemClassifier from ticket
 --
-ALTER TABLE "Ticket" ADD COLUMN "problemClassifier" uuid NULL CONSTRAINT "Ticket_problemClassifier_f1bfcd39_fk_TicketProblemClassifier_id" REFERENCES "TicketProblemClassifier"("id") DEFERRABLE INITIALLY DEFERRED; SET CONSTRAINTS "Ticket_problemClassifier_f1bfcd39_fk_TicketProblemClassifier_id" IMMEDIATE;
+ALTER TABLE "Ticket" ADD COLUMN IF NOT EXISTS "problemClassifier" uuid NULL CONSTRAINT "Ticket_problemClassifier_f1bfcd39_fk_TicketProblemClassifier_id" REFERENCES "TicketProblemClassifier"("id") DEFERRABLE INITIALLY DEFERRED; SET CONSTRAINTS "Ticket_problemClassifier_f1bfcd39_fk_TicketProblemClassifier_id" IMMEDIATE;
 --
 -- Remove field placeClassifier from ticket
 --
-ALTER TABLE "Ticket" ADD COLUMN "placeClassifier" uuid NULL CONSTRAINT "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id" REFERENCES "TicketPlaceClassifier"("id") DEFERRABLE INITIALLY DEFERRED; SET CONSTRAINTS "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id" IMMEDIATE;
+ALTER TABLE "Ticket" ADD COLUMN IF NOT EXISTS "placeClassifier" uuid NULL CONSTRAINT "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id" REFERENCES "TicketPlaceClassifier"("id") DEFERRABLE INITIALLY DEFERRED; SET CONSTRAINTS "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id" IMMEDIATE;
```

Comment:
First command in this line throws an error.
Check whether you already have this constraint locally.